images/.env

```diff
@@ -1,8 +1,8 @@
 FAC_IMG_DEBIAN_TAG=9.13
 FAC_IMG_BASE_TAG=2021-03-10
 FAC_IMG_EPICS_TAG=2021-03-10
-FAC_IMG_APPS_TAG=2021-03-15
-FAC_IMG_IOCS_TAG=2021-03-15
+FAC_IMG_APPS_TAG=2021-03-22
```

Comment:
@fernandohds564 , related to your comment in repo lnls-ansible PR: these lines are being edited because we are not yet using the final version of the ansible makefile targets. we could, as soon sa that PR is merged. Here in this PR, as fr these two lines, I could as well just leave these unedited and update them when we deployed on monday, when creating the new docker image. The process continues to be done somewhat "manually" anyway, before that lnls-ansible PR is not merged.